src/Sylius/Bundle/AdminBundle/Resources/views/Security/ResetPassword/_form.html.twig

```diff
@@ -0,0 +1,16 @@
+{% form_theme form '@SyliusAdmin/Form/theme.html.twig' %}
+
+<div class="ui middle aligned center aligned grid">
+    <div class="column">
+        {{ sylius_template_event('sylius.admin.reset_password.before_form') }}
+
+        {{ form_start(form, {action: path('sylius_admin_password_reset', {'token': app.request.attributes.get('token')}), 'attr': {'class': 'ui large loadable form', 'novalidate': 'novalidate'}}) }}
+            <div class="ui left aligned very padded segment">
+                {{ sylius_template_event('sylius.admin.reset_password.form.content', _context) }}
+            </div>
+            {{ form_row(form._token) }}
+        {{ form_end(form, {'render_rest': false}) }}
+
+        {{ sylius_template_event('sylius.admin.reset_password.after_form') }}
+    </div>
```

Comment:
I would be for adding here only one event with proper blocks, as the form could be extracted to a separate file and used as block defined in yaml
```suggestion
        {{ sylius_template_event('sylius.admin.reset_password.form') }}
    </div>
```